ql/src/java/org/apache/hadoop/hive/ql/exec/repl/DirCopyTask.java

```diff
@@ -140,7 +142,23 @@ public int execute() {
         }
       });
     } catch (Exception e) {
-      throw new SecurityException(ErrorMsg.REPL_RETRY_EXHAUSTED.format(e.getMessage()), e);
```

Comment:
need to check why this task was throwing the exception initially

Reply:
This check is being done in the following lines.